java/ql/lib/semmle/code/java/dataflow/internal/DataFlowNodes.qll

```diff
@@ -44,7 +44,8 @@ newtype TNode =
   } or
   TSummaryInternalNode(SummarizedCallable c, FlowSummaryImpl::Private::SummaryNodeState state) {
     FlowSummaryImpl::Private::summaryNodeRange(c, state)
-  }
+  } or
+  TFieldValueNode(Field f)
```

Comment:
Restrict to static fields?

Reply:
I guess that doesn't work as you also use it in e.g. `DispatchFlow`.

Reply:
Exactly.